### PR TITLE
Add negative test cases

### PR DESCRIPTION
**Pull request by rodion-arr**
- Add more tests for cases that throws exceptions from commands
- Add skip parameter for existing tests that do real request in case no GitHub token found (according to https://github.com/pkgjs/wiby/pull/38#issuecomment-688264931)

### CODE REVIEW

File: test/cli.js [hunk 1]
```diff
@@ -48,4 +48,17 @@ tap.test('result command', async (tap) => {
     tap.equal(true, result.includes("[ 'fake_run', 'queued' ]"))
     tap.equal(true, result.includes("[ 'fake_run_2', 'fake_conclusion' ]"))
   })
+
+  tap.test('result command handles empty response from github.getChecks()', tap => {
+    const result = childProcess.execSync(`${wibyCommand} result --dependent="https://github.com/fakeOrg/fakeRepo"`, {
```
Comment (dominykas) [inline test/cli.js:53]: We should reference our own `wiby-test` here, just in case.

Comment (rodion-arr) [reply test/cli.js:53]: Real requests to GitHub API are mocked in this test anyway

Comment (dominykas) [reply test/cli.js:53]: Yeah, I realize that - that's the "just in case" bit ;)

Comment (rodion-arr) [reply test/cli.js]: done

**Review by dominykas — APPROVED** [review]

**Pull request merged**